python/ray/tests/test_basic.py

```diff
@@ -2992,3 +2992,16 @@ def export_definitions_from_worker(remote_function, actor_class):
         ray.get(actor_handle.method.remote())
 
     ray.get(export_definitions_from_worker.remote(f, Actor))
+
+
+def test_forward(ray_start_cluster):
```

Comment:
this test was used for debugging, no longer needed

Reply:
maybe good to keep it as it is very simple and can be used as a regression test for `forward task` case nicely

Reply:
there's already such a test in `python/ray/tests/perf_integration_tests/test_perf_integration.py`.
you can use that one.

Reply:
ok